.github/workflows/publish.yaml

```diff
@@ -0,0 +1,44 @@
+name: Publish Packages
+
+on:
+  workflow_dispatch:
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18.x'
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: Authenticate to npm
+        run: echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" > ~/.npmrc
+
+      - name: Install dependencies
+        run: yarn install
+
+      - name: Build packages
+        run: yarn run build
+
+      - name: Run tests
+        run: yarn run test
```

Comment:
Can we imagine having e2e test before publishing ?

Reply:
We can in this temporary setup.
The end game would be to trigger this workflow after the CI when there is a merge on the branch `main` (hence after a build, unit tests and E2E).
I'll update this workflow for the moment with the E2E.

Reply:
Done